docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
+
+By default, Docker does not push foreign layers to registries. To
```

Comment:
This really needs some explanation.
e.g. "Some images contain content which is non-distributable due to licensing constraints or other reasons, as an example the Windows base image..."